charts/ccd-case-management-web/values.preview.template.yaml

```diff
@@ -3,20 +3,20 @@ nodejs:
   ingressHost: ${SERVICE_FQDN}
   environment:
     IDAM_LOGIN_URL: https://idam-web-public.aat.platform.hmcts.net
-    CCD_GW_LOGOUT_URL: https://gateway-ccd.aat.platform.hmcts.net/logout
-    CCD_API_URL: https://gateway-ccd.aat.platform.hmcts.net/aggregated
-    CCD_DATA_URL: https://gateway-ccd.aat.platform.hmcts.net/data
-    CCD_GW_OAUTH2_URL: https://gateway-ccd.aat.platform.hmcts.net/oauth2
-    DM_URL: https://gateway-ccd.aat.platform.hmcts.net/documents
+    CCD_GW_LOGOUT_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/logout
+    CCD_API_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/aggregated
+    CCD_DATA_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/data
+    CCD_GW_OAUTH2_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/oauth2
+    DM_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/documents
     DM_URL_REMOTE: ^https?://(?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|dm-store-aat\.service\.core-compute-aat\.internal(?::\d+)?)/documents
-    ANNOTATION_API_URL: https://gateway-ccd.aat.platform.hmcts.net/em-anno
-    PRINT_SERVICE_URL: https://gateway-ccd.aat.platform.hmcts.net/print
+    ANNOTATION_API_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/em-anno
+    PRINT_SERVICE_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/print
     PRINT_SERVICE_URL_REMOTE: https://return-case-doc-ccd.nonprod.platform.hmcts.net
-    CCD_ACTIVITY_URL: https://gateway-ccd.aat.platform.hmcts.net/activity
-    PAYMENTS_URL: https://gateway-ccd.aat.platform.hmcts.net/payments
-    PAY_BULKSCAN_URL: https://gateway-ccd.aat.platform.hmcts.net/pay-bulkscan
-    SHUTTER_REDIRECT_URL: https://xui-webapp-aat.service.core-compute-aat.internal
-    POSTCODE_LOOKUP_URL: https://gateway-ccd.aat.platform.hmcts.net/addresses?postcode={{ .Values.postcodeParam }}
+    CCD_ACTIVITY_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/activity
+    PAYMENTS_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/payments
+    PAY_BULKSCAN_URL: https://ccd-api-gateway-web-pr-353.service.core-compute-preview.internal/pay-bulkscan
+    SHUTTER_REDIRECT_URL: https://xui-webapp-ithc.service.core-compute-ithc.internal
```

Comment:
why change to ithc?

Reply:
for testing purpose I copied the urls values from RDM-12109-off-master to point to pr-353. While doing so I copied this value also. It is not required Mutlu. I will revert it back. Shall I revert back all URL's to point to aat?